.github/workflows/assess-prerelease-compatibility.yml

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023-2025 Northwestern University.
+#
+
+# Make sure to subscribe to notifications
+# See https://docs.github.com/en/actions/concepts/workflows-and-actions/notifications-for-workflow-runs
+# Also note that at time of writing GitHub unfortunately only notifies
+# the last person to have changed the cron schedule
+name: Assess compatibility with pre-release version of InvenioRDM
+
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    # Run every Saturday at 3am UTC
+    - cron: "0 3 * * 6"
```

Comment:
Any reason for Saturday runs? Why not Monday, or during one of the weekdays?

Reply:
No particular reason. I can move it to a weekday since it's at 3am and the following day is a workday during which I could check on any issue.

Reply:
Thank you!